index.html

```diff
@@ -31,14 +31,82 @@ <h3>I’am a Software Developer</h3>
                 <li><img src="pictures/Vector2.png" alt=""></li>
                 <li><img src="pictures/Vector3.png" alt=""></li>
                 <li><img src="pictures/Vector4.png" alt=""></li>
-
             </ul>
-            <!-- <img class="grop" src="pictures/Group.png" alt=""> -->
         </section>
-
-
     </section>
+    <section class="second-main">
+        <h4>My Recent Works</h4>
+        <img src="pictures/line.png" class="line" alt="">
+        <img src="pictures/ImgPlaceholder.png" alt="">
+        <h5>Multi-Post Stories</h5>
+        <p class="par-black">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div>
+            <img src="pictures/Tag.png" alt="">
+            <img src="pictures/Tag(1).png" alt="">
+            <img src="pictures/Tag(2).png" alt="">
+            <img src="pictures/Tag(3).png" alt="">
+        </div>
```

Comment:
- [ ] This project is still missing a `View Project` `<a>` link or `<burron>`. It's also missing a proper `<ul>` `<li>` list of technologies. Please fix it.